src/shapes/audio_token.class.js

```diff
@@ -127,7 +142,13 @@
         'idleImageSrc',
         'selectedImageSrc',
         'pauseControlImageSrc',
-        'playControlImageSrc'].concat(propertiesToInclude));
+        'playControlImageSrc',
+        'hoveredPlayControlImageSrc',
+        'hoveredPauseControlImageSrc',
+        'clickedPlayControlImageSrc',
+        'clickedPauseControlImageSrc',
+        'grayScaleIdleImageSrc',
+        'grayScalePlayControlImageSrc'].concat(propertiesToInclude));
```

Comment:
I think this means the src data (the whole base64 string) will be part of the save data for each audio token which will increase the save file size. 

No changes needed for this now, and I know this is just an extension of the way I was already doing it - so I will investigate, and if so I'll make a ticket to make a system to assign these on load instead of reading the src base64 from the save-file.